src/frontend/src/btc/types/btc-transaction.ts

```diff
@@ -0,0 +1,6 @@
+import type { TransactionType } from '$lib/types/transaction';
+
+export type BtcTransactionType = Exclude<
```

Comment:
Wouldn't `Pick` be more future prone here?

Reply:
Yes, it is less confusing for sure! I was thinking that it was not really top code while coding eheh!

Didn't thought of `Pick`